lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
-<label class="u-field-title" aria-hidden="true">
-    <%- title %>
-</label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<% if (title) { %>
+<span class="u-field-title" aria-hidden="true"><%- title %></span>
```

Comment:
This is really clunky and problematic.  I think we need to add more logic here.  If a form  ```<input>```  (even a disabled one) is on the page, it must have a ```<label>```  element (that is not aria-hidden).  If the value is being simply being displayed (not in a form ```<input>```) then the  ```<label>```  should not be present at all.  When would the screenReaderTitle differ from the title?  Sometimes the title needs some additional context, can we just append or prepend the screenReaderTitle text to the title rather than keep two separate, overlapping titles?